pywbemtools/pywbemcli/_connection_repository.py

```diff
@@ -255,76 +355,83 @@ def _load_connections_file(self):
                        format(B08_DEFAULT_CONNECTIONS_FILE,
                               DEFAULT_CONNECTIONS_FILE))
```

Comment:
Currently, the `IOError` that can happen when renaming fails e.g. due to permissions would cause a traceback, so maybe it should be handled and transformed into `ConnectionFileLoadError`.